src/app/_components/PageWrapper.tsx

```diff
@@ -56,13 +56,54 @@ function WrapperWithBg({ children }: { children: ReactNode }) {
 export function PageWrapper({
   tokenPrice,
   children,
+  statusBarContent,
 }: {
   tokenPrice: TokenPrice;
   children: ReactNode;
+  statusBarContent: IncidentContent;
 }) {
+  const isTestnet = useGlobalContext().activeNetwork.mode === 'testnet';
+  const incidentsToShow = statusBarContent?.items?.filter(
+    alert => (alert.fields.showOnTestnet && isTestnet) || (alert.fields.showOnMainnet && !isTestnet)
+  );
+
+  const statusBarBg = useColorModeValue('black', 'white');
+  const colorMode = useColorMode().colorMode;
+
   return (
     <>
-      <IncidentsStatusBarWithErrorBoundary />
+      <Flex
+        direction={'column'}
+        width={'100%'}
+        top={'0'}
+        backdropFilter={'blur(10px)'}
+        background={statusBarBg}
+      >
+        <IncidentsStatusBarWithErrorBoundary />
```

Comment:
Does this not render the status bar?  It's a little confusing to see this then the status bar base being rendered below

Reply:
I agree, will restructure this part.